join.go

```diff
@@ -23,7 +39,15 @@ func join(t joinType, table, on interface{}) Builder {
 		buf.WriteString("JOIN ")
 		switch table := table.(type) {
 		case string:
-			buf.WriteString(d.QuoteIdent(table))
+			table = strings.Replace(table, " AS ", " as ", 1)
```

Comment:
there is helper function 'As', which allows to create aliases. Can it be used for same purpose?
If not, I would prefer to add new builder for table alias instead of parsing table name.

Reply:
As doesn't alias tables in the Join clause, unfortunately.